### DISCUSSION TITLE
Can't build android code

### DISCUSSION THREAD

**Discussion by newfuture33** [strongswan/strongswan#625] [General]
![image](https://user-images.githubusercontent.com/34529419/134701555-5cef1c1b-3768-4f4a-94f9-4b588d710227.png)
I am very urgent. please help me

**Comment by tobiasbrunner** [comment]
It seems you are trying to build the Android app on Windows. I'm not sure if that's possible (see e.g. [my comment to an old issue](https://wiki.strongswan.org/issues/2234#note-1), even though you might have a more basic problem with `ndk-build` here).
